### ISSUE TITLE
own data?

### ISSUE THREAD

**Issue by VerstraeteP** [mitroadmaps/roadtagger#4]
Can you provide a brief explanation of how to use own images, to predict road lanes? So use pretrained model to predict lanes in whole new image

thanks in advance

**Comment by songtaohe** [comment]
So far there is no such feature in the code. I will try to add a standalone inference module (very likely a docker container) to this repo. It may take a few days to work on it. I will let you know once it is done.

**Comment by songtaohe** [comment]
Hi, I add a RoadTagger [docker container](https://github.com/mitroadmaps/roadtagger/tree/master/docker) --- it should support inference on custom images. Please let me know if there are any issues.

**Comment by VerstraeteP** [comment]
Thanks a lot!
few questions: What is the graph file? is this just the start and end coordinates of the piece of road you want to know?
And how do you obtain the ground sampling distance of an image, is there some formula? Because now i don't get the desired results.

thanks!

**Comment by songtaohe** [comment]
@VerstraeteP The distribution of the lane count is not balanced. I was trying to give you a concrete number about the distribution of the lane count in the training dataset, but unfortunately, we deleted the training dataset on our server because we were low in disk space. Indeed, we find our model is not doing well on some very obvious roads. It could be caused by the training dataset we used. As you know, we create the training dataset using OpenStreetMap data from 20 US cities. The labels from OSM are not always correct. Sometimes they are inconsistent in different places. We are thinking about working on a new version of roadtagger that can take the label quality issue into account.
